packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -187,27 +188,37 @@ export class TablePageObject<T extends TableRecord> {
         return anchor as Anchor;
     }
 
-    public getRenderedIconColumnCellIcon(
+    public getRenderedIconColumnCellContent(
         rowIndex: number,
         columnIndex: number
-    ): Icon {
-        const icon = this.getRenderedCellView(rowIndex, columnIndex).shadowRoot!
-            .firstElementChild;
-        if (!icon || !(icon instanceof Icon)) {
+    ): Icon | Spinner {
+        const content = this.getRenderedCellView(rowIndex, columnIndex)
+            .shadowRoot!.firstElementChild;
+        if (
+            !content
+            || !(content instanceof Icon || content instanceof Spinner)
+        ) {
             throw new Error(
-                `Icon not found at cell ${rowIndex},${columnIndex}`
+                `Icon or Spinner not found at cell ${rowIndex},${columnIndex}`
             );
         }
-        return icon;
+        return content;
     }
 
-    public getRenderedIconColumnGroupHeaderIcon(groupRowIndex: number): Icon {
-        const icon = this.getGroupRowHeaderView(groupRowIndex).shadowRoot!
+    public getRenderedIconColumnGroupHeaderContent(
```

Comment:
This isn't returning the whole group header content. So I'm wondering if the name should be updated to indicate that it'll just return the icon/spinner, or if the function should be updated to return an object with `text: string` and `notSureWhatToNameThis: Icon | Spinner`. Perhaps the question is whether or not there is another function on the page object that would let you separately get the text, or if you have to return it from here to have it be available to tests?

Reply:
There is already a function that returns the text content, but it was named `getRenderedGroupHeaderContent`. I decided to rename four functions to make them more specific about their behavior:
`getRenderedGroupHeaderContent` -> `getRenderedGroupHeaderTextContent`
`getRenderedCellContent` -> `getRenderedCellTextContent`
`getRenderedIconColumnGroupHeaderContent` -> `getRenderedIconColumnGroupHeaderContentElement`
`getRenderedIconColumnCellContent` -> `getRenderedIconColumnCellContentElement`